src/main.py

```diff
@@ -95,28 +95,30 @@ def checkRunSettings(settings, components):
   if 'NPV_search' in settings.getIndicators() and sum(comp.countMulttargets() for comp in components) < 1:
     raise IOError('NPV_search in <Indicators> "name" but no cash flows have "mult_target=True"!')
 
-def checkDrivers(settings, components, variables, v=100):
+def checkDrivers(settings, components, variables, v=100, pyomoDriv=False):
   """
     checks if all drivers needed are present in variables
     @ In, settings, CashFlows.GlobalSettings, global settings
     @ In, components, list, list of CashFlows.Component instances
     @ In, variables, dict, variable-value map from RAVEN
     @ In, v, int, verbosity level
+    @ In, pyomoDriv, boolean, 'True' will trigger pyomo flag
```

Comment:
similar comment here, let's explain what the flag means for the sake of future dev

Reply:
I think this is an outstanding need still.